cli/Valet/Site.php

```diff
@@ -724,7 +725,9 @@ function proxyCreate($url, $host)
         }
 
         $siteConf = $this->replaceOldLoopbackWithNew(
-            $this->files->get(__DIR__.'/../stubs/proxy.valet.conf'),
+            $this->files->get(
+                $unsecure ? __DIR__.'/../stubs/proxy.valet.conf' : __DIR__.'/../stubs/secure.proxy.valet.conf'
```

Comment:
I wonder, would it make sense to name this `insecure.proxy.valet.conf` to make it super clear?